.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
 sudo: false
 node_js:
-  - 6
+  - 6.11.1
 env:
   - NODE_ENV="test"
+before_script:
+  - yarn run db:reset
+  - yarn run db:setup:user
 script:
-  - npm test
-  - npm run enforce
-  - npm run lint
+  - yarn run enforce
+  - yarn run lint
+  - yarn test
```

Comment:
I think we jumped the gun on having you alphabetize here -- Travis runs each script command sequentially so long as the previous one doesn't fail. See (here)[https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step]. 

BUT, we don't even have Travis enabled as of now, so you'll need to log in with github at the Travis-ci (homepage)[https://travis-ci.com/] and add this repo so that we actually have tests in this PR.

THEN, you'll want to comment out the `yarn run enforce` line because your tests are going to fail, since you have no tests yet. Then everything should be good! 👍

Reply:
Oh shoot, my bad.